rafthttp/transport.go

```diff
@@ -78,13 +81,24 @@ type Transporter interface {
 	Stop()
 }
 
-type transport struct {
-	roundTripper http.RoundTripper
-	id           types.ID
-	clusterID    types.ID
-	raft         Raft
-	serverStats  *stats.ServerStats
-	leaderStats  *stats.LeaderStats
+// Transport is the implementation of Transporter. It provides the functionality
+// to send raft messages to peers, and receive raft messages from peers.
+// The Handler in Transport needs to serve requests sent to peer URLs of the
+// local member for remote Transport to work well.
+// Transport needs to call Start before calling other functions, and call
+// Stop when it is no longer used.
+type Transport struct {
+	RoundTripper http.RoundTripper  // roundTripper to send requests
+	ID           types.ID           // local member ID
+	ClusterID    types.ID           // raft cluster ID for request validation
+	Raft         Raft               // raft state machine, to which the Transport processes received messages and reports status
```

Comment:
Transport processes received message to raft state machine? The grammar seems to be wrong.